dogfooding/lib/screens/home_screen.dart

```diff
@@ -75,6 +78,56 @@ class _HomeScreenState extends State<HomeScreen> {
     }
   }
 
+  Future<void> _directCall(BuildContext context) async {
```

Comment:
Do we want to add dogfooding changes in the same PR?

Reply:
hmm good question. On one side it would be nice to have a core SKD change as a separate PR on the other side we can't test it without the dogfooding changes ;)

Reply:
Let's merge it for now mainly because push is notoriously difficult to test on for now. Keep in mind someone might ask us to remove this in the future to adhere to the design and we may need to comply.

Reply:
Also, this is a unique example but in the future let's merge these separately since we can revert specific commits much more easily.